server/src/modules/task/task.controller.ts

```diff
@@ -17,15 +17,15 @@ export class TaskController {
 
   @Get('all-task')
   async findAll(): Promise<TaskInterface[]> {
-    return this.taskService.getAllTask();
+    return this.taskService.getAllTasks();
   }
 
   @Get(':id')
   async getTaskById(@Param('id') id: number): Promise<Task> {
     return this.taskService.getTaskById(id);
   }
 
-  @Post('create')
+  @Post('task')
```

Comment:
When will creating, link gonna  be `***/task/task`?

Reply:
This should be changed to something else, but not to task.

Reply:
You could just do `@Post('')`, so the link will be POST `***/task/`